arrow/example_test.go

```diff
@@ -0,0 +1,62 @@
+// Run Tarantool instance before example execution:
+//
+// Terminal 1:
+// $ cd arrow
+// $ TEST_TNT_LISTEN=3013 TEST_TNT_WORK_DIR=$(mktemp -d -t 'tarantool.XXX') tarantool config.lua
```

Comment:
config.lua creates a space with the `memtx` engine. Why do a user need the example with a failed request? It's better to add a working example with Tarantool EE..